dnf-behave-tests/dnf/environment.py

```diff
@@ -265,12 +269,15 @@ def before_scenario(context, scenario):
         if skip:
             scenario.skip(reason="Disabled by default by @xfail.")
 
-    if context.tag_matcher.should_exclude_with(scenario.effective_tags):
+    if context.os_tag_matcher.should_exclude_with(scenario.effective_tags):
         scenario.skip(reason="Disabled by OS version tag.")
 
+    if context.dnf_tag_matcher.should_exclude_with(scenario.effective_tags):
+        scenario.skip(reason="Disabled by active tag.")
```

Comment:
I'd make the message more explicit by saying e.g. _"Disabled by dnf = 4/5 tag."_